src/vue/modules/issuance-form/mixins/get-receiver-account.mixin.spec.js

```diff
@@ -0,0 +1,102 @@
+import GetReceiverAccountMixin from './get-receiver-account.mixin'
+
+import { Wallet } from '@tokend/js-sdk'
+
+import { errors } from '@/js/errors'
+
+import { mount, createLocalVue } from '@vue/test-utils'
+
+import * as Api from '../_api'
+
+const localVue = createLocalVue()
+
+const Component = {
+  template: '<div></div>',
+}
+
+describe('Get receiver account mixin', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(Component, {
+      mixins: [GetReceiverAccountMixin],
+      localVue,
+    })
+  })
+
+  describe('method', () => {
+    beforeEach(() => {
+      const wallet = new Wallet(
+        'test@mail.com',
+        'SCPIPHBIMPBMGN65SDGCLMRN6XYGEV7WD44AIDO7HGEYJUNDKNKEGVYE',
+        'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ',
+        '4aadcd4eb44bb845d828c45dbd68d5d1196c3a182b08cd22f05c56fcf15b153c'
+      )
+      const config = {
+        horizonURL: 'https://test.api.com',
+      }
+
+      Api.initApi(wallet, config)
+    })
```

Comment:
you do not need to `initApi` with the real data

Reply:
Fixed